projects/plugins/jetpack/modules/search/instant-search/components/search-result.jsx

```diff
@@ -49,7 +53,15 @@ class SearchResult extends Component {
 		if ( this.props.resultFormat === RESULT_FORMAT_PRODUCT ) {
 			return <SearchResultProduct onClick={ this.onClick } { ...this.props } />;
 		} else if ( this.props.resultFormat === RESULT_FORMAT_EXPANDED ) {
-			return <SearchResultExpanded onClick={ this.onClick } { ...this.props } />;
+			const isMultiSite =
+				this.props.staticFilters && this.props.staticFilters.group_id !== MULTISITE_NO_GROUP_VALUE;
```

Comment:
I'm seeing non-P2 WPCOM sites rendering with `isMultiSite === true`. I also see this happening on P2 sites even when I'm not included in the `P2_EARLY_ADOPTERS` user list.

Reply:
Good catch, should be fixed now.